app/src/Application.php

```diff
@@ -81,24 +86,8 @@ public function bootstrap(): void
     public function middleware(MiddlewareQueue $middlewareQueue): MiddlewareQueue
     {
         $middlewareQueue
-            // Catch any exceptions in the lower layers,
-            // and make an error page/response
-            ->add(new ErrorHandlerMiddleware(Configure::read('Error'), $this))
-
-            // Handle plugin/theme assets like CakePHP normally does.
-            ->add(new AssetMiddleware([
-                'cacheTime' => Configure::read('Asset.cacheTime'),
-            ]))
```

Comment:
I think I had commented before about removing this LOC was NG. However, I can't find the comment anymore. 😭 

Both middlewares by CakePHP are important. Removing it will have unintended side effect. 

Please kindly add again the ErrorHandlerMiddleware & AssetMiddleware.

Reply:
> I think I had commented before about removing this LOC was NG. However, I can't find the comment anymore. 😭

Found it https://github.com/ArielGalvez/tutorial-cakephp/pull/4#discussion_r1377521267. Can you explain why it's happening again in this PR?

Reply:
This an interesting question, I didn't realized that, probably I lost those changes when rebasing the previous branches, in this case from the authentication. I solved them fast appliying only the current. But now is restaured